client/src/app/app.component.html

```diff
@@ -1,31 +1,31 @@
 <mat-sidenav-container class="sidenav-container">
-  <mat-sidenav #drawer class="sidenav" fixedInViewport mode="over">
-    <mat-toolbar>Menu</mat-toolbar>
-    <mat-nav-list class="sidenav-list">
+  <mat-sidenav #drawer class="sidenav" fixedInViewport [autoFocus]="false">
+    <mat-toolbar>Client</mat-toolbar>
+    <mat-nav-list class="sidenav-list" role="navigation">
 
       <!-- Side navigation links -->
 
-      <mat-list-item routerLink="/" routerLinkActive="drawer-list-item-active"
+      <a mat-list-item routerLink="/" routerLinkActive="drawer-list-item-active"
       #routerLinkActiveInstance="routerLinkActive"
       [attr.tabindex]="routerLinkActiveInstance.isActive ? 0 : -1"
       [routerLinkActiveOptions]="{exact: true}" (click)="drawer.close()">
       <mat-icon matListIcon>home</mat-icon>
       Home
```

Comment:
Just a quick indentation fix. This one predates your PR, but I thought it would be good to include it anyway! :)
```suggestion
        <mat-icon matListIcon>home</mat-icon>
        Home
```

Reply:
Wow the indentation in that entire file is a total mess. I went and fixed it in a new commit.